src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/DatePickerViewModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Windows.Input;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Data;
+
+namespace SamplesApp.Windows_UI_Xaml_Controls.Models
+{
+	[Bindable]
+	public class DatePickerViewModel : ViewModelBase
+	{
+		private DateTime _Date = DateTime.Now.Date;
```

Comment:
```suggestion
		private DateTime _date = DateTime.Now.Date;
```
to follow guidelines

Reply:
Done